packages/components/collapsible-panel/src/collapsible-panel.tsx

```diff
@@ -259,7 +266,10 @@ const CollapsiblePanel = (props: TCollapsiblePanel) => {
   );
 };
 
-CollapsiblePanel.getPanelContentId = getPanelContentId;
+/**
+ * @deprecated This function is no longer supported.
+ */
+CollapsiblePanel.getPanelContentId = () => '';
```

Comment:
I think this function [was introduced](https://github.com/commercetools/ui-kit/pull/1272/files#diff-8d9b9a02ab995f407cfe27c55cc92198345216e51dfbac565109a13f4b28fa55) when writing the first tests that took a11y into account but I updated them to not rely on it since it was also not working in every scenario.

Although this function is public, I don't think it's being actually used.
I search the Merchant Center frontend and App-kit repositories and didn't find any usage of this function.

I'm not sure if it would be ok to just remove it but it could happen that any `ui-kit` user could be using it. But again, It think that function was not meant to be publicly used not work in all scenarios.

Let me know what you think about it.